styles.css

```diff
@@ -9,6 +9,7 @@ body{
 }
 header, footer, aside, .main{
   padding: 1em;
+  margin: 0;
```

Comment:
You need to add `margin: 0` to the direct children of these elements, not the elements themselves. Your selector(s) would look something like this, using `header` as an example...

```css
header > * {
  margin: 0;
}
```